dll/win32/shell32/folders/CRegFolder.cpp

```diff
@@ -734,7 +738,7 @@ HRESULT WINAPI CRegFolder::GetDefaultColumn(DWORD dwRes, ULONG *pSort, ULONG *pD
 
 HRESULT WINAPI CRegFolder::GetDefaultColumnState(UINT iColumn, DWORD *pcsFlags)
```

Comment:
DWORD --> SHCOLSTATEF ?

Reply:
CFSFolder also incorrectly uses DWORD, perhaps early ROS headers used DWORD causing the implementations to use DWORD?

Reply:
probably